docs/cosmicrays.rst

```diff
@@ -109,27 +86,48 @@ This will update the file in place, as the flag is set: :code:`inplace=True`.  S
    :width: 600
    :alt: Example for cosmic ray flagging and interpolation from convolution from a Laplacian kernel.
 
-   Example of cosmic-ray flagging from convolution from a Laplacian kernel and bilinear 
-   interpolation to highlight the differences.
+   Example of cosmic-ray flagging from convolution from a Laplacian kernel and bilinear interpolation to highlight the differences.
 
       
 AstroDrizzle Cosmic Ray flagging
 --------------------------------
 
-Coming Soon.
+Since CRs are stochastic events and are uncorrelated between adjacent exposures, then comparing successive images to some combination from those images (such as average or median) will show pixels that deviate by more than some threshold.  However, the WFSS images are often dithered with respect to each other and have a non-negligible amount of image distortion, making the naive comparison impossible.  The standard technique with :term:`direct imaging` is to use the ``AstroDrizzle`` package from the `drizzlepac software <https://drizzlepac.readthedocs.io/en/latest/>`_ to correct for the image dithers and distortion, apply the cosmic-ray flagging thresholds, and produce an stacked image can be extended to WFSS data.  However, the WFSS data brings a new challenge, namely the :doc:`sky background <background>` is not uniform, and AstroDrizzle cannot address these backgrounds.  Therefore, the sky background must be removed before AstroDrizzle can be considered.
```

Comment:
```suggestion
Since CRs are stochastic events and are uncorrelated between adjacent exposures, comparing successive images to some combination from those images (such as average or median) will show pixels that deviate by more than some threshold.  However, the WFSS images are often dithered with respect to each other and have a non-negligible amount of image distortion, making the naive comparison impossible.  The standard technique with :term:`direct imaging` is to use the ``AstroDrizzle`` package from the `drizzlepac software <https://drizzlepac.readthedocs.io/en/latest/>`_ to correct for the image dithers and distortion, apply the cosmic ray flagging thresholds, and produce a stacked image can be extended to WFSS data.  However, the WFSS data brings a new challenge, namely the :doc:`sky background <background>` is not uniform, and AstroDrizzle cannot address these backgrounds.  Therefore, the sky background must be removed before AstroDrizzle can be used.
```